assets/js/payment-method-extensions/payment-methods/stripe/credit-card/use-checkout-subscriptions.js

```diff
@@ -62,7 +62,9 @@ export const useCheckoutSubscriptions = (
 			const type = event.error.type;
 			const code = event.error.code || '';
 			let message = getErrorMessageForTypeAndCode( type, code );
-			message = message || event.error.message;
+			if ( message === null ) {
+				message = event.error.message;
+			}
```

Comment:
This could probably be simplified by using the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). I've implemented in 89e40e0 so this pull can be merged.